### ISSUE TITLE
The meaning of 'NUM_TRAIN_SAMPLES', 'NUM_TEST_SAMPLES' and the question of the discriminator.

### ISSUE THREAD

**Issue by XiaobenLi00** [shubham-goel/4D-Humans#70]
Thanks a lot for your great work!
I have two questions.
1. I notice 'NUM_TRAIN_SAMPLES' and 'NUM_TEST_SAMPLES' in the config file and thety seems to be used in the model, but I am not sure what do they mean, could you provide more details?
2. When training HMR2, the adverserial loss is also used as in HMR, but I notice that you use different hyper-parameters, e.g., learning rate of the discriminator, the weight of the losses. So I am wondering the performance of different settings. In my understanding, for the unpaired setting of HMR, the adverserial loss is critical, but for HMR2, there are pseudo SMPL annotations for all training data, so the adverserial loss plays a different (maybe less important) role, right? I try to use the config of HMR2 for the unpaired setting, but it not seems to work well. I'd appreciate if you could give me some advice.
